certified-connectors/World Academia/readme.md

```diff
@@ -0,0 +1,52 @@
+# Title : World Academia
+The primary purpose of this connector is to provide easy and efficient access to university-related data for educational institutions, researchers, students, and anyone interested in information about universities worldwide. 
+
+## Publisher: Kelcho Tech​
+
+## Prerequisites
+The tool is free to use no authentication or licenses is required.​
+
+## Supported Operations
+The connector will enable users to retrieve data such as lists of universities, university details, continents, countries, and more
+### /api/v1/continentsdata
+Get all continents and their metadata
+
+### /api/v1/continents
+Get a list continents
+
+## /api/v1/continent/{continentName}
+Get a list countries in a continent.​
+
+## /api/v1/sch
+Get all universities and their metadata
+
+## /api/v1/sch/count
+Get the number of  universities available
+
+## /api/v1/sch/countries
+Get all universities and their metadata
+
+## /api/v1/sch/countriescodes
+Get a list of countriesCodes with universities
+
+## /api/v1/sch/country/{countryName}
+Get all universities in a country via country name
+
+## /api/v1/sch/countrycode/{countryCode}
+Get all universities in a country via country code
+
+## /api/v1/schlist
+Get a list of all universities
+
+## /api/v1/sch/university/{universityName}
+Get all university details via university name
+
+## Obtaining Credentials
+The is no authentication needed to access this connector​
+
+## Getting Started
+Optional. How to get started with your connector.
+
+## Known Issues and Limitations
+There is no any known issues, this connector currently works fine.
+
```

Comment:
Please add deployment instructions, please refer this link. https://github.com/microsoft/PowerPlatformConnectors/blob/dev/templates/certified-connectors/readme.md

Reply:
Hello @vmanoharas Please review my changes, I have added the deployment instructions